src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/inject/Injector.kt

```diff
@@ -4,13 +4,12 @@ import org.wycliffeassociates.otter.jvm.device.audio.injection.DaggerAudioCompon
 import org.wycliffeassociates.otter.jvm.device.audioplugin.injection.DaggerAudioPluginComponent
 import org.wycliffeassociates.otter.jvm.persistence.injection.DaggerPersistenceComponent
 import org.wycliffeassociates.otter.jvm.persistence.repositories.*
-import org.wycliffeassociates.otter.jvm.persistence.repositories.mapping.CollectionMapper
 import org.wycliffeassociates.otter.jvm.persistence.repositories.mapping.LanguageMapper
 import org.wycliffeassociates.otter.jvm.persistence.repositories.mapping.ResourceMetadataMapper
 
 object Injector {
     private val persistenceComponent = DaggerPersistenceComponent.builder().build()
-    private val database = persistenceComponent.injectDatabase()
+    val database = persistenceComponent.injectDatabase()
```

Comment:
does database need to be used in a public way?

Reply:
nope, thanks